app/bundles/LeadBundle/Form/Type/ListType.php

```diff
@@ -57,6 +58,12 @@ public function __construct(MauticFactory $factory)
             $this->listChoices[$list['id']] = $list['name'];
         }
 
+        $emails                = $listModel->getUserEmails();
+        $this->emailChoices     = array();
+        foreach ($emails as $email) {
```

Comment:
I'm not sure about this, but if there aren't any emails created yet (a fresh Mautic install), wouldn't the `$emails === NULL`? Because if it would, it would cause a PHP error.